android/java/org/chromium/chrome/browser/playlist/PlaylistServiceObserverImpl.java

```diff
@@ -37,6 +39,9 @@ public void onMediaFileDownloadProgressed(String id, long totalBytes, long recei
                 id, totalBytes, receivedBytes, percentComplete, timeRemaining);
     }
 
+    @Override
```

Comment:
We need to handle this callback in a different branch for android specific changes.